python/cudf/cudf/core/column/column.py

```diff
@@ -1475,7 +1475,7 @@ def build_categorical_column(
     size: int = None,
     offset: int = 0,
     null_count: int = None,
-    ordered: bool = None,
+    ordered: bool = False,
```

Comment:
Oh, please update the docstring below to indicate the parameter is optional and the default is False.

Reply:
Done